src/crypto/BUILD.gn

```diff
@@ -186,6 +222,14 @@ static_library("crypto") {
     ]
   }
 
+  if (chip_with_trusty_os) {
+    sources += [ "PersistentStorageOperationalKeystoreTrusty.cpp" ]
+  } else if (chip_with_imx_ele) {
+    sources += [ "PersistentStorageOperationalKeystoreEle.cpp" ]
+  } else {
+    sources += [ "PersistentStorageOperationalKeystore.cpp" ]
```

Comment:
The point of PersistentStorageOperationalKeystore is that it's a simple keystore that just stores the keys in a kvs-like thing, no?

Other keystores can be injected by someone who wants to use them, but they just shouldn't pretend to be called PersistentStorageOperationalKeystore....